src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -327,6 +329,84 @@ def test_gen_datatype_exception_2():
 #             "variable 'dummy'." in caplog.text)
 
 
+def test_reverse_map():
```

Comment:
Well done adding this 2 tests that should already have been here :)

Reply:
:-)